src/components/MenuContent/FeaturedSection.tsx

```diff
@@ -44,19 +50,47 @@ const FeaturedSection: React.FC<{
               }
             : {};
 
+        const testAPITitle = 'Data API';
+        const testAPIDescription =
+          'Get realtime, local data including daily new cases, % vaccinated, and more.';
+
         return (
           <RowWithSpacing onClick={() => onClick(cta)} key={cta}>
             <IconWrapper>
               <Icon height="36" width="40" aria-hidden="true" />
             </IconWrapper>
             <TextLink to={url} {...hashlinkProps}>
-              <Column>
-                <Row>
-                  {cta}
-                  <ArrowIcon />
-                </Row>
-                <FeaturedDescription>{description}</FeaturedDescription>
-              </Column>
+              {iconId === 'API' && (
+                <Column>
+                  <Experiment id={ExperimentID.HAMBURGER_MENU_DESKTOP}>
+                    <Variant id={VariantID.A}>
+                      <Row>
+                        {cta}
+                        <ArrowIcon />
+                      </Row>
+                      <FeaturedDescription>{description}</FeaturedDescription>
+                    </Variant>
+                    <Variant id={VariantID.B}>
+                      <Row>
+                        {testAPITitle}
+                        <ArrowIcon />
+                      </Row>
+                      <FeaturedDescription>
+                        {testAPIDescription}
+                      </FeaturedDescription>
+                    </Variant>
+                  </Experiment>
+                </Column>
+              )}
+              {iconId !== 'API' && (
+                <Column>
+                  <Row>
+                    {cta}
+                    <ArrowIcon />
+                  </Row>
+                  <FeaturedDescription>{description}</FeaturedDescription>
+                </Column>
+              )}
```

Comment:
looks great! a few small things-
- you can move `Column` outside of the experiment wrapper
- instead of hard-coding the icon ID, you can use the `SectionId` enum, so `SectionId.API` instead of `'API'`.
- also, instead of having 2 separate `&&` clauses, you can combine them into one ternary with `?` and `:`. so:

 ```
{iconId === SectionId.API ? (
                  <Experiment id={ExperimentID.HAMBURGER_MENU_DESKTOP}>
                    <Variant id={VariantID.A}>
                      <Row>
                        {cta}
                        <ArrowIcon />
                      </Row>
                      <FeaturedDescription>
                        {description}
                      </FeaturedDescription>
                    </Variant>
                    <Variant id={VariantID.B}>
                      <Row>
                        CTA 2
                        <ArrowIcon />
                      </Row>
                      <FeaturedDescription>
                        Variation B description
                      </FeaturedDescription>
                    </Variant>
                  </Experiment>
                ) : (
                  <>
                    <Row>
                      {cta}
                      <ArrowIcon />
                    </Row>
                    <FeaturedDescription>{description}</FeaturedDescription>
                  </>
                )}
```